docs/my-website/docs/proxy/logging.md

```diff
@@ -61,9 +61,9 @@ litellm_settings:
 
 Removes any field with `user_api_key_*` from metadata.
 
-## What gets logged? StandardLoggingPayload
```

Comment:
there are many changes made to this doc - can you please scope the doc changes just to adding langtrace